src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,49 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+const setId = id => `#${id}`;
+
+export const Tabs = ({
+  tabs,
+  onTabSelected,
+  selectedTabId,
+  table,
+}) => {
+  const checkChange = (tab) => {
+    if (tab.id !== selectedTabId) {
+      onTabSelected(tab || tabs[0]);
```

Comment:
```suggestion
      onTabSelected(tab);
```
looks like redundant `|| tabs[0]`.